sections/csgcn/csgcn-chapter.tex

```diff
@@ -332,7 +332,7 @@ \subsection{Adjacency matrix}\label{subsec:csgcn_adj_adj_mat}
     \end{bmatrix}
 \end{equation}
 Let the embedding matrix for the 0th layer be $Emb^{(0)} \in \mathbb{R}^{(|U| + |I| + |US| + |IS| + |C|) \times K}$, where $K$ is the embedding size and $C$ is the set of possible context values.
-The embedding matrix for the next layer after $l$ for CSGCN-ADJ is obtained by:
+The embedding matrix for the next layer after layer $l$ for CSGCN-ADJ is obtained by:
```

Comment:
er lill l konsistent med det ovenover hvor jeg kommenterede på forskellige l

Reply:
Det tror jeg det er men måske vi lige skal have en trello opgave med det. Jeg laver en for at kigge det igennem